portfolio/src/main/webapp/script.js

```diff
@@ -50,6 +59,20 @@ async function getComments() {
   });
 } /* eslint-enable no-unused-vars */
 
+/**
+ * Get login status of user from json
```

Comment:
nit: either in this line, or in the @param, you can clarify what you mean by "json". You can explain that you're taking a JSON object that contains login information about the current user, obtained from the /login-status URL.